Source/ZXing.Net.Mobile.iOS/MobileBarcodeScanner.cs

```diff
@@ -190,11 +191,11 @@ public override void Cancel ()
 				((UIViewController)viewController).InvokeOnMainThread(() => {
 					viewController.Cancel();
 
-                    // Calling with animated:true here will result in a blank screen when the scanner is closed on iOS 7.
-                    ((UIViewController)viewController).DismissViewController(false, null); 
+					// Calling with animated:true here will result in a blank screen when the scanner is closed on iOS 7.
+					((UIViewController)viewController).DismissViewController(is7orgreater, null); 
```

Comment:
Sorry, should have changed the comment. On iOS7 value of is7orgreater should be false and I've just looked at my check in the Initialise method and it's wrong, should be:
```
is7orgreater = sv.Major > 7;
```